src/khoj/routers/api.py

```diff
@@ -862,3 +862,8 @@ async def extract_references_and_questions(
         compiled_references = [item.additional["compiled"] for item in result_list]
 
     return compiled_references, inferred_queries, defiltered_query
+
+    # health check
```

Comment:
nit: can you remove this comment? the api definition is pretty self-explanatory 🙂 .